packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -324,3 +337,423 @@ describe('Error handling - prop types', () => {
     expect(console.error).toHaveBeenCalledWith(expect.stringContaining(ERRORS.ROW_HEADER_INDEX_EXCEEDS_PINNED)); // eslint-disable-line no-console
   });
 });
+
+describe('Row selection', () => {
+  let mockOnCellSelect = null;
+  let mockOnRowSelect = null;
+  let mockOnEnableRowSelection = null;
+
+  beforeEach(() => {
+    mockOnCellSelect = jest.fn();
+    mockOnRowSelect = jest.fn();
+    mockOnEnableRowSelection = jest.fn();
+  });
```

Comment:
Is `beforeEach` necessary? I would think that you need this only once in a `beforeAll` block?

Reply:
Apparently, this doesn't work in `beforeAll`, disregard this.